src/pkg/quota/types/format_test.go

```diff
@@ -15,7 +15,6 @@
 package types
 
 import "testing"
-
```

Comment:
same comment as this one:
* https://github.com/goharbor/harbor/pull/17131/files#r923010380